specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json

```diff
@@ -702,6 +755,40 @@
     }
   },
   "definitions": {
+    "HostQuota": {
+      "description": "Host quotas",
+      "properties": {
+        "gp": {
+          "description": "VS20 host quota",
+          "type": "integer"
+        },
+        "he": {
+          "description": "VS36 host quota",
+          "type": "integer"
+        }
+      }
+    },
+    "QuotaGet": {
+      "description": "Subscription quotas",
+      "properties": {
+        "hostsRemaining": {
+          "$ref": "#/definitions/HostQuota"
+        },
+        "quotaEnabled": {
+          "description": "Subscription has a valid quota",
```

Comment:
>Subscription has a valid quota [](start = 26, length = 30)

what does having a valid quota mean?  It isn't clear from the name or description